scalableminds/webknossos
# Fix rendering for intensity range of 0

This PR fixes a bug where having one layer with the intensity range of 0 led to no data at all being rendered.

This problem is tackled by 
1. determining `min == max`
1. if `min == max` add +1 to the divisor of the division to avoid dividing by 0
1. Later replace the calculated color value with black in the case of `min == max`.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Please do the following steps for a few datasets to ensure this fix works for "all". I tested this on the ROI and l4 sample dataset.
- Open a dataset in view mode
- Go to the layer settings and move the slider of a color layer of your choice to the same values (e.g. 100).
- The data layer should now no longer be rendered but the other active layers should still be visible.
- Additionally, there should be a warning icon stating that the layer is not rendered and that the user should prefer to disable the layer instead of reducing the intensity range to 0.

### Issues:
- fixes #5671

------
